packages/app/src/server/routes/api/index.ts

```diff
@@ -26,7 +26,7 @@ function api(): express.Router {
   const api = express.Router();
   // mount auth handlers
   api.use("/auth", auth());
-  api.use("/public-ui-config", pubUiCfg());
+  api.use("/public-ui-config", pubUiCfg);
```

Comment:
I see the easy mistake here - being mislead by the existing naming convention of the other handlers. They'd be better name as `getAuthHandler()` for example.

Reply:
BTW, have you been running and testing this locally or pushing to CI and waiting for the result there?

Reply:
> have you been running and testing this locally or pushing to CI and waiting for the result there?

Yesterday I did that, yes. The linting and compile stages in CI are rather useful for getting good feedback.

Today I have set up the local dev environment with the help of your docs -- thanks!

Reply:
> being mislead by the existing naming convention of the other handlers

Right : ).

Pushing a commit for improving that.